tools/oeedger8r/src/Sources.ml

```diff
@@ -178,7 +178,10 @@ let rec get_ptr_setter get_deepcopy args count setter (ptype, decl) =
         (* NOTE: This makes the embedded check in the `OE_` macro superfluous. *)
         [
           sprintf "if (pargs_in->%s)"
-            (String.concat " && pargs_in->" (List.rev (arg :: args)));
+            (if args = [] then arg
```

Comment:
Do we support arbitrarily nested structures?

Reply:
Current deepcopy logic is that given a first-level pointer, try recursively to find all the nested pointers of that pointer.